src/docfx/config/Config.cs

```diff
@@ -302,7 +303,7 @@ internal class Config : PreloadConfig
         /// <summary>
         /// Determines and configures build to consume XML files produced from monodoc
         /// </summary>
-        public ECMA2YamlRepoConfig? Monodoc { get; private set; }
+        public ECMA2YamlRepoConfig[]? Monodoc { get; private set; }
```

Comment:
`OneOrManyConverter`?

Reply:
Cool, ideally it should be.
(I thought that currently this value was handled by `OpsConfigAdapter`, and it's definitely an `array`...)